crates/telio-model/src/features.rs

```diff
@@ -428,6 +429,13 @@ where
     Ok(Some(eps))
 }
 
+/// Customize firewall policy
+#[derive(Default, Copy, Clone, Debug, PartialEq, Eq, Deserialize)]
+pub struct FeatureFirewall {
+    /// Custom private IP range
+    pub custom_private_ip_range: Option<Ipv4Network>,
```

Comment:
This was added particularly to pass integration tests because `stun_server` uses a private IP hence its packets are dropped. Hence, a vector doesn't make sense at this moment.